packages/react/src/components/Footer/README.md

```diff
@@ -0,0 +1,56 @@
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Footer } from '@ibmdotcom/react';
```

Comment:
@photodow we should probably instruct users to point to the es folder if they want to take advantage of tree-shaking. I provided both as an example in the JSDocs for services and utilities:
https://deploy-preview-55--ibmdotcom-services.netlify.com/searchtypeaheadapi